src/main/java/uk/gov/ons/census/casesvc/messaging/UndeliveredMailReceiver.java

```diff
@@ -79,7 +79,7 @@ public void receiveMessage(Message<ResponseManagementEvent> message) {
   }
 
   private boolean shouldEmitCaseUpdated(Case caze) {
-    return !caze.isRefusalReceived()
+    return StringUtils.isEmpty(caze.getRefusalReceived())
```

Comment:
Left over `StringUtils.isEmpty` check, should be `caze.getRefusalReceived() == null`